templates/default/cfnconfig.erb

```diff
@@ -1,22 +1,22 @@
-stack_name=<%= node['cluster']['stack_name'] %>
-preinstall=<%= node['cluster']['preinstall'] %>
-preinstall_args=(<%= node['cluster']['preinstall_args'] %>)
-postinstall=<%= node['cluster']['postinstall'] %>
-postinstall_args=(<%= node['cluster']['postinstall_args'] %>)
-region=<%= node['cluster']['region'] %>
-scheduler=<%= node['cluster']['scheduler'] %>
-scheduler_slots=<%= node['cluster']['scheduler_slots'] %>
-instance_slots=<%= node['cluster']['instance_slots'] %>
-ephemeral_dir=<%= node['cluster']['ephemeral_dir'] %>
-ebs_shared_dirs=<%= node['cluster']['ebs_shared_dirs'] %>
-proxy=<%= node['cluster']['proxy'] %>
-node_type=<%= node['cluster']['node_type'] %>
-cluster_user=<%= node['cluster']['cluster_user'] %>
+cfn_stack_name=<%= node['cluster']['stack_name'] %>
+cfn_preinstall=<%= node['cluster']['preinstall'] %>
+cfn_preinstall_args=(<%= node['cluster']['preinstall_args'] %>)
+cfn_postinstall=<%= node['cluster']['postinstall'] %>
+cfn_postinstall_args=(<%= node['cluster']['postinstall_args'] %>)
+cfn_region=<%= node['cluster']['region'] %>
+cfn_scheduler=<%= node['cluster']['scheduler'] %>
+cfn_scheduler_slots=<%= node['cluster']['scheduler_slots'] %>
+cfn_instance_slots=<%= node['cluster']['instance_slots'] %>
+cfn_ephemeral_dir=<%= node['cluster']['ephemeral_dir'] %>
+cfn_ebs_shared_dirs=<%= node['cluster']['ebs_shared_dirs'] %>
+cfn_proxy=<%= node['cluster']['proxy'] %>
+cfn_node_type=<%= node['cluster']['node_type'] %>
+cfn_cluster_user=<%= node['cluster']['cluster_user'] %>
 <% if node['cluster']['node_type'] == 'ComputeFleet' -%>
-head_node=<%= node['cluster']['head_node'] %>
-head_node_private_ip=<%= node['cluster']['head_node_private_ip'] %>
-scheduler_queue_name=<%= node['cluster']['scheduler_queue_name'] %>
+  cfn_head_node=<%= node['cluster']['head_node'] %>
+  cfn_head_node_private_ip=<%= node['cluster']['head_node_private_ip'] %>
+  cfn_scheduler_queue_name=<%= node['cluster']['scheduler_queue_name'] %>
 <% end -%>
 <% if node['cluster']['node_type'] == 'HeadNode' -%>
-volume=<%= node['cluster']['volume'] %>
+  cfn_volume=<%= node['cluster']['volume'] %>
```

Comment:
is it safe to indent these variables?

Reply:
Fixed here: https://github.com/aws/aws-parallelcluster-cookbook/pull/1102